frontend/src/components/whaleopedia-page/WhaleCard.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from "react";
+import {
+  getAllSpecies,
+  ListResponse,
+  Species,
+  ConservationStatus,
+} from "../../clients/apiClient";
+import "./Whaleopedia.scss";
+
+interface WhaleCardProps {
+  whale: Species;
+}
+
+export const WhaleCard: React.FC<WhaleCardProps> = ({ whale }) => {
+  return (
+    <div className="whale-list_card">
+      <div className="whale-list_card--common-name">{whale.name}</div>
+      <img className="whale-list_card--image" src={whale.photoUrl} />
+      <div className="whale-list_card--scientific">
```

Comment:
Very very good use of BEM naming here! Only one note - these are Elements of the Block (which is a `whale-card`), not Modifiers, so you should be using double underscores for them:
```suggestion
      <div className="whale-card__common-name">{whale.name}</div>
      <img className="whale-card__image" src={whale.photoUrl} />
      <div className="whale-card__scientific">
```